pkg/controller/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -546,6 +548,29 @@ func TestDeleteMultipleUserAccountsViaMasterUserRecordBeingDeleted(t *testing.T)
 		DoesNotHaveFinalizer()
 }
 
+func TestDisablingMasterUserRecord(t *testing.T) {
+	// given
+	logf.SetLogger(logf.ZapLogger(true))
+	s := apiScheme(t)
+	mur := murtest.NewMasterUserRecord("john", murtest.DisabledMur(true))
+	userAccount := uatest.NewUserAccountFromMur(mur)
```

Comment:
be aware that the `disabled` field is copied from MUR, which means that you initialize the test with `UserAccount.disabled == true`, which means that you don't test anything here.